compiler/crates/relay-typegen/src/typescript.rs

```diff
@@ -258,6 +261,34 @@ impl TypeScriptPrinter {
                         )?;
                         self.write_indentation()?;
                     }
+                    if self.generate_jsdoc {
+                        let has_doc = key_value_pair.doc_comment.is_some()
+                            || key_value_pair.deprecation.is_some();
+                        if has_doc {
+                            write!(&mut self.result, "/**\n")?;
+                        }
+                        if key_value_pair.doc_comment.is_some() {
+                            self.write_indentation()?;
+                            write!(
+                                &mut self.result,
+                                "* {}\n",
+                                key_value_pair.doc_comment.as_ref().unwrap()
+                            )?;
+                        }
+                        if key_value_pair.deprecation.is_some() {
+                            self.write_indentation()?;
+                            write!(
+                                &mut self.result,
+                                "* @deprecated {}\n",
+                                key_value_pair.deprecation.as_ref().unwrap()
```

Comment:
Can be a followup task, but should we include the deprecation reason?

Reply:
mmmh maybe i'm wrong, but there's already the reason? 
or you mean we shouldn't include it?

Reply:
Oh sorry. Not sure how I missed that.